Cargo.lock

```diff
@@ -13,6 +13,15 @@ dependencies = [
  "cpufeatures",
```

Comment:
This file seems to have more changes than I expect. Can you revert it back and then let Cargo make only the necessary changes?

Reply:
will do.

Reply:
Done